# Start_Here
### Navigation links (and brief descriptions) to my repositories.

* [Start_Here](https://github.com/pwilmart/Start_Here) - This repository

* [pwilmart.github.io](https://github.com/pwilmart/pwilmart.github.io) - Account website repository

---

## Software

* [PAW_pipeline](https://github.com/pwilmart/PAW_pipeline) - The PAW/Comet proteomics pipeline

* [fasta_utilities](https://github.com/pwilmart/fasta_utilities) - Utilities for downloading and prepping FASTA files

* [utilities](https://github.com/pwilmart/utilities) - Some miscellaneous utility scripts

* [annotations](https://github.com/pwilmart/annotations) - Scripts for adding UniProt annotations to results lists

* [PAW_BLAST](https://github.com/pwilmart/PAW_BLAST) - Scripts for BLAST ortholog matching

* [Z-score_GUI](https://github.com/pwilmart/Z-score_GUI) - Script for sliding-window Z-score analyses

---

## Analyses

* [PXD017823_Real-Time-Search](https://github.com/pwilmart/PXD017823_Real-Time-Search) - Analysis of some real time search data

#### Internal Reference Scaling

* [IRS_validation](https://github.com/pwilmart/IRS_validation) - Validation and background for the Internal Reference Scaling method

* [IRS_normalization](https://github.com/pwilmart/IRS_normalization) - Example of the IRS method

* [Plubell_2017_PAW](https://github.com/pwilmart/Plubell_2017_PAW) - Reanalysis of original IRS paper's data

#### PAW TMT analyses

* [MaxQuant_and_PAW](https://github.com/pwilmart/MaxQuant_and_PAW) - Comparison of PAW and MaxQuant with same TMT data

* [Multiple_TMT_MQ](https://github.com/pwilmart/Multiple_TMT_MQ) - Example of IRS method with MaxQuant data

* [Dilution_series](https://github.com/pwilmart/Dilution_series) - Comparison of PSM, peptide, and protein TMT data

#### Other TMT analyses

* [TMT_analysis_examples](https://github.com/pwilmart/TMT_analysis_examples) - Descriptions of various TMT analyses

* [Yeast_CarbonSources](https://github.com/pwilmart/Yeast_CarbonSources) - TMT data from yeast grown with different sugars

* [Yeast_triple_KO_TMT](https://github.com/pwilmart/Yeast_triple_KO_TMT) - Yeast triple knockout data

* [PXD001077_P-furiosus](https://github.com/pwilmart/PXD001077_P-furiosus) - Not yet completed

* [Metaplastic-BC_PXD014414](https://github.com/pwilmart/Metaplastic-BC_PXD014414) - Reanalysis of a 3-plex, 27-sample cancer study demonstrating IRS

* [BCP-ALL_QE-TMT_Nat-Comm-2019](https://github.com/pwilmart/BCP-ALL_QE-TMT_Nat-Comm-2019) - A 3-plex, 27-sample MS2-based TMT study of cancer data

#### MS2 Reporter Ions

* [SPS-MS3_vs_MS2_TMT](https://github.com/pwilmart/SPS-MS3_vs_MS2_TMT) - Comparison of MS2 TMT and SPS MS3 TMT for same data

* [JPR-201712_MS2-MS3](https://github.com/pwilmart/JPR-201712_MS2-MS3) - A comparison of MS2 and SPS MS3 data for an E. coli background

#### Spectral Counting

* [Smith_SpC_2018](https://github.com/pwilmart/Smith_SpC_2018) - Quantitative analysis of large SpC dataset

* [Sea_lion_urine_SpC](https://github.com/pwilmart/Sea_lion_urine_SpC) - Analysis of sea lion urine samples

* [ABRF_iPRG_2015_SpC](https://github.com/pwilmart/ABRF_iPRG_2015_SpC) - Analysis of ABRF iPRG data from 2015 study

---

## Meetings

* [talk_to_repo_example](https://github.com/pwilmart/talk_to_repo_example) - How to turn meeting content into repositories

* [ABRF_2020](https://github.com/pwilmart/ABRF_2020) - How to use GitHub to support scientific research

* [ASMS-2013_search-engine-comparison](https://github.com/pwilmart/ASMS-2013_search-engine-comparison) - How to compare search engines

* [ASMS_2018](https://github.com/pwilmart/ASMS_2018) - The Internal Reference Scaling (IRS) method

* [Cascadia_2013](https://github.com/pwilmart/Cascadia_2013) - Extended parsimony protein grouping

* [Cascadia_2018](https://github.com/pwilmart/Cascadia_2018) - A tour of Jupyter notebooks

* [RECOMB-CP_2012](https://github.com/pwilmart/RECOMB-CP_2012) - How to analyze shotgun proteomics data

---

## Forked Repositories

* [Reference_Proteome_Manager](https://github.com/pwilmart/Reference_Proteome_Manager)

* [tmt-normalization](https://github.com/pwilmart/tmt-normalization)

* [spectrum_utils](https://github.com/pwilmart/spectrum_utils)

* [RawQuant](https://github.com/pwilmart/RawQuant)

* [RawTools](https://github.com/pwilmart/RawTools)

* [stats337](https://github.com/pwilmart/stats337)
